final_prep/mod_zero_hero.rb

```diff
@@ -1,40 +1,66 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
+hero_name = 'The Flash'
+special_ability = 'Super Speed'
 
 # Declare two variables - greeting AND catchphrase
-#   greeting should be assigned to a string that uses interpolation to include the hero_name
-#   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "I'm #{hero_name}. I'm the fastest man alive."
+catchphrase = "You can't keep up with me. I've got #{special_ability}"
 
 # Declare two variables - power AND energy - set to integers
+power = 200
+energy = 50
 
 # Declare two variables - full_power AND full_energy
-#   full_power should multiply your current power by 500
-#   full_energy should add 150 to your current energy
+full_power = power * 500
+full_energy = energy + 150
 
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
+arch_enemies = ['Captain Cold', 'Gorila Grodd', 'Reverse Flash']
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+sidekicks = ['Kid Flash', 'Impulse', 'Wally West']
 # Print the first sidekick to your terminal
-
+p sidekicks.first
 # Print the last arch_enemy to the terminal
-
+p arch_enemies.last
 # Write some code to add a new arch_enemy to the arch_enemies array
+arch_enemies.push('Mirror Master')
 
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
+p arch_enemies
 
 # Remove the first sidekick from the sidekicks array
+sidekicks.shift
 
 # Print the sidekicks array to terminal to ensure you added a new sidekick
+p sidekicks
 
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
+def assess_situation(danger_level, save_the_day, bad_excuse)
+  if danger_level > 50
+    p bad_excuse
+  elsif danger_level === (10..50)
```

Comment:
If danger_level is 25, this line of code will return `false`.

You should not be using the `===` here. This means something totally different than `==` (which asserts equality).